### PR TITLE
Add flake to repository

### PR DESCRIPTION
**Pull request by TheNeikos**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  outputs = { self, nixpkgs, crane, flake-utils, rust-overlay, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ (import rust-overlay) ];
+        };
+
+        rustTarget = pkgs.rust-bin.stable."1.60.0".default;
```
Comment (matthiasbeyer) [inline flake.nix:30]: I'd prefer to pull this from a `rust-toolchain` file, would that be possible?

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

**Comment by matthiasbeyer** [comment]
CI will fail before #77 is merged anyways, so I won't trigger it for now.

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
Feel free to rebase and then merge.

bors delegate=TheNeikos

**Comment by TheNeikos** [comment]
bors r+

Local nix flake check built fine!

**Pull request merged**